### DISCUSSION TITLE
Curso FullStack.NET - Request e Response

### DISCUSSION THREAD

**Discussion by mkydouglas** [balta-io/forum#576] [Tira Dúvidas] [answered]
Olá, Balta!

Na aula o senhor menciona que uma requisição GET não tem corpo. No caso de uma requisição apenas de consulta com vários filtros o correto seria fazer um GET passando os filtros por parâmetro?

Geralmente aqui na empresa o pessoal faz essa consulta enviando um POST com os filtros no corpo. Na sua visão qual é a melhor abordagem?

Muito obrigado!!

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @mkydouglas, como vai?

Ótima pergunta... se olharmos para as definições exatas, nos mínimos detalhes, o `GET` (Obter) seria o recomendado, mas eu concordo que para passar N parâmetros o `POST` pode ser melhor (e mais fácil).

Então eu não acho que seja errado... eu tomaria o `GET` como primeira opção, mas caso necessário fazer um `POST` para obter os dados, acho OK também.
